src/Orleans.Reminders/Orleans.Reminders.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Orleans.Reminders</PackageId>
     <Title>Microsoft Orleans Reminders Library</Title>
-    <Description>Reminders library for Microsoft Orleans used both on the server.</Description>
+    <Description>Reminders library for Microsoft Orleans used both on the client and server.</Description>
```

Comment:
Reminders are not available from the client, they are only available from within grains.